modules/theme-tools/compat/twentytwentyone.css

```
@@ -0,0 +1,9 @@
+/**
+ * Related Posts
+ */
+
+#jp-relatedposts {
```

I see previous theme compat files have done something along the lines of
```suggestion
.entry-content #jp-relatedposts {
```
to get the CSS priority right.

---

👍 I was just committing that. c25659c24120864fbe63a95dec151642bf143fb7